src/languages/singlestoredb/singlestoredb.keywords.ts

```diff
@@ -0,0 +1,852 @@
+    'SETOF',
+    'SECURITY_LISTS_INTERSECT',
+    'SHA',
+    'SHARD',
+    'SHARDED',
+    'SHARDED_ID',
+    'SHARE',
+    'SHOW',
+    'SHUTDOWN',
+    'SIGNAL',
+    'SIGNED',
+    'SIMILAR',
+    'SIMPLE',
+    'SITE',
+    'SKIP',
+    'SKIPPED_BATCHES',
+    '__SLEEP',
+    'SMALLINT',
+    'SNAPSHOT',
+    '_SNAPSHOT',
+    '_SNAPSHOTS',
+    'SOFT_CPU_LIMIT_PERCENTAGE',
+    'SOME',
+    'SONAME',
+    'SPARSE',
+    'SPATIAL',
+    'SPATIAL_CHECK_INDEX',
+    'SPECIFIC',
+    'SQL',
+    'SQL_BIG_RESULT',
+    'SQL_BUFFER_RESULT',
+    'SQL_CACHE',
+    'SQL_CALC_FOUND_ROWS',
+    'SQLEXCEPTION',
+    'SQL_MODE',
+    'SQL_NO_CACHE',
+    'SQL_NO_LOGGING',
+    'SQL_SMALL_RESULT',
+    'SQLSTATE',
+    'SQLWARNING',
+    'STDIN',
+    'STDOUT',
+    'STOP',
+    'STORAGE',
+    'STRAIGHT_JOIN',
+    'STRICT',
+    'STRING',
+    'STRIP',
+    'SUCCESS',
+    'SUPER',
+    'SYMMETRIC',
+    'SYNC_SNAPSHOT',
+    'SYNC',
+    '_SYNC',
+    '_SYNC2',
+    '_SYNC_PARTITIONS',
+    '_SYNC_SNAPSHOT',
+    'SYNCHRONIZE',
+    'SYSID',
+    'SYSTEM',
+    'TABLE',
+    'TABLE_CHECKSUM',
+    'TABLES',
+    'TABLESPACE',
+    'TAGS',
+    'TARGET_SIZE',
+    'TASK',
+    'TEMP',
+    'TEMPLATE',
+    'TEMPORARY',
+    'TEMPTABLE',
+    '_TERM_BUMP',
+    'TERMINATE',
+    'TERMINATED',
+    'TEXT',
+    'THEN',
+    'TIME',
+    'TIMEOUT',
+    'TIMESTAMP',
+    'TIMESTAMPADD',
+    'TIMESTAMPDIFF',
+    'TIMEZONE',
+    'TINYBLOB',
+    'TINYINT',
+    'TINYTEXT',
+    'TO',
+    'TRACELOGS',
+    'TRADITIONAL',
+    'TRAILING',
+    'TRANSFORM',
+    'TRANSACTION',
+    '_TRANSACTIONS_EXPERIMENTAL',
+    'TREAT',
+    'TRIGGER',
+    'TRIGGERS',
+    'TRUE',
+    'TRUNC',
+    'TRUNCATE',
+    'TRUSTED',
+    'TWO_PHASE',
+    '_TWOPCID',
+    'TYPE',
+    'TYPES',
+    'UNBOUNDED',
+    'UNCOMMITTED',
+    'UNDEFINED',
+    'UNDO',
+    'UNENCRYPTED',
+    'UNENFORCED',
+    'UNHOLD',
+    'UNICODE',
+    'UNION',
+    'UNIQUE',
+    '_UNITTEST',
+    'UNKNOWN',
+    'UNLISTEN',
+    '_UNLOAD',
+    'UNLOCK',
+    'UNLOGGED',
+    'UNPIVOT',
+    'UNSIGNED',
+    'UNTIL',
+    'UPDATE',
+    'UPGRADE',
+    'USAGE',
+    'USE',
+    'USER',
+    'USERS',
+    'USING',
+    'UTC_DATE',
+    'UTC_TIME',
+    'UTC_TIMESTAMP',
+    '_UTF8',
+    'VACUUM',
+    'VALID',
+    'VALIDATE',
+    'VALIDATOR',
+    'VALUE',
+    'VALUES',
+    'VARBINARY',
+    'VARCHAR',
+    'VARCHARACTER',
+    'VARIABLES',
+    'VARIADIC',
+    'VARYING',
+    'VERBOSE',
+    'VIEW',
+    'VOID',
+    'VOLATILE',
+    'VOTING',
+    'WAIT',
+    '_WAKE',
+    'WARNINGS',
+    'WEEK',
+    'WHEN',
+    'WHERE',
+    'WHILE',
+    'WHITESPACE',
+    'WINDOW',
+    'WITH',
+    'WITHOUT',
+    'WITHIN',
+    '_WM_HEARTBEAT',
+    'WORK',
+    'WORKLOAD',
+    'WRAPPER',
+    'WRITE',
+    'XACT_ID',
+    'XOR',
+    'YEAR',
+    'YEAR_MONTH',
+    'YES',
+    'ZEROFILL',
+    'ZONE',
+  ],
+  constraints: ['ON DELETE', 'ON UPDATE'],
+  charset: ['CHARACTER SET'],
```

Comment:
These two lines should be moved inside the `*.formatter.ts` file.

This part was changed recently. See the `mysql.formatter.ts` file for reference or the PR #363